src/pocketmine/block/BlockIds.php

```diff
@@ -41,7 +41,7 @@
 	const IRON_ORE = 15;
 	const COAL_ORE = 16;
 	const LOG = 17, WOOD = 17, TRUNK = 17;
-	const LEAVES = 18;
+	const LEAVES = 18, LEAVE = 18;
```

Comment:
1. This is not the valid singular form of leaves.
2. I doubt anyone would want to use `LEAF` as the block name... I really doubt whether we should put so many different forms, just to make the Item::fromString() method more powerful, while it isn't needed.